lessons/AI/ai-linear-regression/langs/en-us/index.adoc

```diff
@@ -0,0 +1,223 @@
+= Artificial Intelligence &  Regression
+
+@description{Students learn about the world’s first self-driving car. They then apply and extend their prior knowledge of functions and linear regression, using multiple regression to make predictions.}
+
+@ifproglang{pyret}{
+@lesson-prereqs{ai-training}
+}
+
+
+@keywords{}
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+@objectives
+
+
+
+| Student-facing Lesson Goals
+|
+
+- Let's learn about the functioning of the very first self-driving car!
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+== Supervised Learning @duration{15 minutes}
+
+@objective{regression-ml}
+@objective{car-function}
+
+=== Overview
+Students learn about ALVINN (autonomous Land Vehicle in a Neural Network) as an example of supervised machine learning and regression.
+
+=== Launch
+
+@right{@image{images/navlab.png, 400}}
+
+Meet ALVINN (pictured to the right).
+
+ALVINN (short for Autonomous Land Vehicle in a Neural Network) was developed in 1989 by a team of computer scientists at Carnegie Mellon University. ALVINN is the forebear of today's self-driving cars: a retrofitted Army ambulance equipped with sensors, computers, and actuators (the part of a machine that allows movement). ALVINN had about one-tenth of the processing power of an Apple Watch - but was able to safely drive at 70 miles per hour (albeit on a very closed course) by the early 1990s! How did it happen?
+
+As with all machine learning, ALVINN required training.
+
+The goal of that training? Learning how to steer.
+
+Here's what ALVINN's training entailed:
+
+- A person drove the vehicle while a front-facing video camera recorded the road.
+- Every two seconds, ALVINN digitized an image of the road ahead and recorded the chosen steering direction.
+- Training images were reduced in resolution to 30x32 pixels and provided as inputs associated with the corresponding "correct" steering angles as their outputs.
+
+The human driver in this context is often referred to as the _supervisor_; it is their tagging of input-output pairs that makes supervised machine learning possible.
+
+Once ALVINN's training data has been collected, it can be represented with a table, as below.
+
+@teacher{
+Note: These images were selected for illustrative purposes. The 30x32 pixel images produced by ALVINN were of much lower quality!
+}
+
+[cols="^.^1,^.^1", stripes="none", options="header"]
+|===
+
+| x (input data) | y (output data)
+| @image{images/road1.png, 100} 	| 0.02º
+| @image{images/road2.png, 100} 	| -95.2º
+| @image{images/road3.png, 100} 	| 135.43º
+
+|===
+
```

Comment:
could add an optional activity here where we provided some images of the road and had students estimate what angle ALVINN should steer?